src/edu/asu/plp/tool/prototype/model/PLPProject.java

```diff
@@ -145,8 +147,13 @@ public PLPProject(String name)
 	@Override
 	public void save()
 	{
-		// TODO: implement
-		throw new UnsupportedOperationException("Not Yet Implemented");
+		File file = new File(this.getPath());
+		file.mkdirs();
+		System.out.println(this.getPath());
+		System.out.println(this.getName());
+		//System.out.println(this.getFileCount());
+
+		//throw new UnsupportedOperationException("Not Yet Implemented");
```

Comment:
See below. This code:
```
// TODO: implement
throw new UnsupportedOperationException("Not Yet Implemented");
```
should not be deleted until the method is fully implemented.
If (for some reason) the method was implemented, but is broken, then the TODO should be replaced with something along the lines of:
```
// FIXME: [describe what's wrong]
```